<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 252, 207);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_calculate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(77, 146, 235);
border: 3px solid gray;
color: white;
}
QPushButton:pressed {
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                    stop: 0 rgb(71, 133, 214), stop: 1 rgb(77, 146, 235));
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>621</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ежемесячный платеж</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="butt_overpayment">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="monthly_payment">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Общая выплата</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Переплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="total_payment">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="type_calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>207</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Расчет ежемесячного платежа</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>106</y>
     <width>613</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Сумма кредита</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="credit_sum">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string>0.01</string>
      </property>
      <property name="maxLength">
       <number>11</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>руб.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Срок кредита</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="credit_period">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="combo_credit_period">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <item>
       <property name="text">
        <string>лет</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>месяцев</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="max_period">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 47, 47);
border: 3px solid gray;
color: white;</string>
      </property>
      <property name="text">
       <string>Макс. значение 50. Миним. значение 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="interest_rate">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="text">
       <string>0.01</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="max_rate">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 47, 47);
border: 3px solid gray;
color: white;</string>
      </property>
      <property name="text">
       <string>Макс. значение 999. Миним. значение 0.01</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Тип ежемесячных платежей</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="type_payment">
      <item>
       <property name="text">
        <string>Аннуитетные</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дифференцированные</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Вариант расчета</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
